sync-files-local.js

```diff
@@ -20,7 +21,7 @@ const PROJECTS_DIR = path.dirname(ROOT_DIR); // Parent directory containing all
 
 // Get repos from environment variable or load from config
 const reposToSync = process.env.REPOS
-  ? process.env.REPOS.split(",").map((r) => r.trim())
+  ? process.env.REPOS.split(",").map((r) => `macpro-mdct-${r.trim()}`)
```

Comment:
This was a tiny bug I found while testing

Reply:
Thank you!